internal/extproc/backendauth/aws_test.go

```diff
@@ -0,0 +1,43 @@
+package backendauth
+
+import (
+	"testing"
+
+	corev3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	extprocv3 "github.com/envoyproxy/go-control-plane/envoy/service/ext_proc/v3"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewAWSHandler(t *testing.T) {
+	t.Setenv("AWS_ACCESS_KEY_ID", "test")
+	t.Setenv("AWS_SECRET_ACCESS_KEY", "secret")
+
+	handler, err := newAWSHandler(nil)
+	require.NoError(t, err)
+	require.NotNil(t, handler)
+}
+
+func TestAWSHandler_Do(t *testing.T) {
+	t.Setenv("AWS_ACCESS_KEY_ID", "test")
+	t.Setenv("AWS_SECRET_ACCESS_KEY", "secret")
```

Comment:
Can we support credential file as well like aws filter? We need to refresh the aws credential when it expires. @aabchoo can move his cred refreshing code here.

Reply:
Yes of course. We will do after #43